src/main/java/com/toptrumps/db/FullStats.java

```diff
@@ -0,0 +1,61 @@
+package com.toptrumps.db;
+import java.sql.*;
+import java.util.*;
+import com.toptrumps.core.player.Player;
+
+public class FullStats{
+
+    public FullStats(ResultSet rs,PlayersList players) throws SQLException{
+
+        this.players = players;
+        this.gameID = rs.getInt(1);
+        this.winnerID = rs.getInt(2);
+        this.roundsPlayed = rs.getInt(3);
+        this.draws = rs.getInt(4);
```

Comment:
Nathan, I think it is better to use rs.getInt with column names, not column indexes. It is more readable and more reliable (in case of different columns order). What do you think?